### ISSUE TITLE
b'value 0for Parameter num_class should be greater equal to 1'

### ISSUE THREAD

**Issue by zhangjiahuan17** [kingfengji/gcForest#58]
I'm trying to use this model to do classification, but I only 2 classes.
Here comes the errors:
1. when I only change the` ca_config["n_classes"]=2`, the error raised like this `b'value 0for Parameter num_class should be greater equal to 1' `
2. Then I add `"num_class"=2` into estimators, it raised a new error `__init__() got an unexpected keyword argument 'num_class'`
I don't what should I do....Is there anyone who can help me, please....?

**Comment by GuoshengCui** [comment]
How do you solve this problem in the end? I encountered the same problem......

**Comment by 550911204** [comment]
I just add "num_class:  2" in the tail of the config. like this   
ca_config["estimators"].append(
            {"n_folds": 5, "type": "XGBClassifier", "n_estimators": 10, "max_depth": 5,
             "objective": "multi:softprob", "silent": True, "nthread": -1, "learning_rate": 0.1, "num_class": 2
Then it seems this error disappear.And the code run succussfully, however the accuracy if 0.00%

Emmmmmm.....Who can help me?

**Comment by jaegerstar** [comment]
@550911204 You should change the "objective": "multi:softprob" to "objective": "binary:logistic".
